rpcs3/rpcs3qt/main_window.cpp

```diff
@@ -2388,7 +2385,17 @@ void main_window::keyPressEvent(QKeyEvent *keyEvent)
 	{
 		switch (keyEvent->key())
 		{
-		case Qt::Key_E: if (Emu.IsPaused()) Emu.Resume(); else if (Emu.IsReady()) Emu.Run(true); return;
+		case Qt::Key_E:
+		{
+			switch (Emu.GetStatus())
+			{
+			case system_state::paused: Emu.Resume(); return;
```

Comment:
wrong order

Reply:
Wdym? this is an equivalent of previous code.

Reply:
Ah. I confused it with IsPaused(), which now also means is_ready.